tests/contrast/min_contrasts.json

```diff
@@ -18,6 +18,7 @@
   "gruvbox-light": 3.2,
   "igor": 3.6,
   "inkpot": 1.0,
+  "lightbulb": 3.9,
```

Comment:
updated minimum contrast to pass test

Reply:
That doesn't really solve the accessibility problem. Is there any chance you can actually get the theme above the WCAG recommended values? WCAG recommends 4.5, and this may be down due to a few colors only.